packages/tables/frontend/sirius-components-tables/src/table/useTableColumns.tsx

```diff
@@ -68,7 +68,16 @@ export const useTableColumns = (
     const rowHeaderColumn: MRT_ColumnDef<GQLLine, string> = {
       id: 'mrt-row-header',
       header: '',
+      enableResizing: enableColumnSizing,
       columnDefType: 'display',
+      muiTableHeadCellProps: {
+        sx: {
+          ' .Mui-TableHeadCell-Content': {
+            height: '100%',
+            paddingBottom: '0.6rem',
```

Comment:
This is just another way to hardcode some specific values, I just want things to be centered. I'll look into it